system-test/fixtures/google-gax-packaging-test-app/src/index.js

```diff
@@ -40,18 +40,38 @@ if (require.main === module) {
 }
 
 async function testShowcase() {
-  const client = new gapic.v1beta1.EchoClient({
+  const authStub = {
+    getRequestHeaders() {
+      return {Authorization: 'Bearer SOME_TOKEN'};
+    },
+  };
+
+  const grpcClientOpts = {
     grpc,
     sslCreds: grpc.credentials.createInsecure(),
-  });
+  };
+
+  const fallbackClientOpts = {
+    auth: authStub,
+    fallback: true,
+    port: 1337,
+  };
+
+  const grpcClient = new gapic.v1beta1.EchoClient(grpcClientOpts);
+
+  const fallbackClient = new gapic.v1beta1.EchoClient(fallbackClientOpts);
 
   // assuming gRPC server is started locally
-  await testEcho(client);
-  await testExpand(client);
-  await testPagedExpand(client);
-  await testCollect(client);
-  await testChat(client);
-  await testWait(client);
+  await testEcho(grpcClient);
+  await testExpand(grpcClient);
+  await testPagedExpand(grpcClient);
+  await testCollect(grpcClient);
+  await testChat(grpcClient);
+  await testWait(grpcClient);
+
+  await testEcho(fallbackClient);
```

Comment:
We might want to mention here that streaming calls aren't currently supported when using gRPC-fallback. Or alternately have tests making sure an exception is thrown when we try to use streaming calls.

Reply:
Making sure an exception is thrown is a great idea, since it (a) checks the intended behavior, and (b) explains itself clearly.  Since we don't use `chai` in this file, just regular 

```js
try {
  await stuff();
  throw new Error("stuff is supposed to throw because it's not supported");
}
catch (err) {
  // ignoring the error
}
```
should work and show the idea.

Reply:
Good idea, done.